src/traccuracy/_tracking_graph.py

```diff
@@ -147,7 +147,15 @@ def __init__(
             name (str, optional): User specified name that will be included in result
                 outputs associated with this object
         """
+
+        # if the segmentation is not integer dtype, cast to uint64. Otherwise, leave it
```

Comment:
Comment needs to be updated. Unfortunately I can't figure out how to do a suggestion spanning multiple lines